### PR TITLE
Add section on services to data model section.

### PR DESCRIPTION
**Pull request by msporny**
Add initial content to the data model section on services.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/pull/32.html" title="Last updated on Oct 3, 2019, 9:01 PM UTC (120f022)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-spec/32/0c41898...120f022.html" title="Last updated on Oct 3, 2019, 9:01 PM UTC (120f022)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -531,6 +531,16 @@ <h2>
       <h2>
 Services
       </h2>
+      <p>
+Service descriptions are used in DID Documents to describe ways in which one can
```
Comment (dlongley) [inline index.html]:
```suggestion
Service descriptions are used in DID Documents to express ways in which one can
```

File: index.html [hunk 2]
```diff
@@ -531,6 +531,16 @@ <h2>
...
+communicate with the subject of the DID or associated entities. Services listed
+in the DID Document can contain information about privacy preserving messaging
+services, or more public information such as social media accounts, personal
+websites, and email addresses. The metadata  associated with services are often
```
Comment (dlongley) [inline index.html]:
```suggestion
websites, and email addresses. The metadata associated with services are often
```

File: index.html [hunk 3]
```diff
@@ -531,6 +531,16 @@ <h2>
...
+service specific. For example, an encrypted messaging service most likley also
```
Comment (dlongley) [inline index.html]: The encrypted messaging service doesn't "contain" this information -- rather the metadata might express it, right? Suggest:

```
For example, the metadata associated with an encrypted messaging service most likely expresses how to initiate the encrypted link before messaging begins.
```

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (msporny) [reply index.html]: Fixed.

File: index.html (hunk 3 above)
Comment (msporny) [reply index.html]: Fixed.

**Comment by msporny** [comment]
Multiple reviewers, applied suggestions, no objections, merging.

**Pull request merged**